src/client/datascience/notebook/kernelWithMetadata.ts

```diff
@@ -22,6 +22,9 @@ export class VSCodeNotebookKernelMetadata implements VSCNotebookKernel {
             Uri.file(join(this.context.extensionPath, 'out', 'ipywidgets', 'dist', 'ipywidgets.js')),
             Uri.file(
                 join(this.context.extensionPath, 'out', 'datascience-ui', 'ipywidgetsKernel', 'ipywidgetsKernel.js')
+            ),
+            Uri.file(
+                join(this.context.extensionPath, 'out', 'datascience-ui', 'ipywidgetsKernel', 'fontAwesomeLoader.js')
```

Comment:
Not sure why `fontAwesomeLoader.js`1 needs to be in the `ipywidgetKernel` folder.
This sends the wrong message, i'd move it else where like `notebook`